dom/ipc/ContentParent.cpp

```diff
@@ -916,10 +916,13 @@ already_AddRefed<ContentParent> ContentParent::GetUsedBrowserProcess(
   // new tabs to not render. There might be a simple fix here but it doesn't
   // seem worth investigating.
   if (aRecordingDispatchAddress.Length() > 0) {
-      return GetUsedBrowserProcessForRecording(
-        aRemoteType, aContentParents, aRecordingDispatchAddress);
+    return GetUsedBrowserProcessForRecording(
+      aRemoteType, aContentParents, aRecordingDispatchAddress);
   }
 
+  return nullptr;
```

Comment:
This should have a comment, can you move the one above the if statement here, since it doesn't apply to the if statement anymore?